source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRContentStackView.mm

```diff
@@ -150,6 +150,31 @@ - (void)addArrangedSubview:(UIView *)view
     [self.stackView addArrangedSubview:view];
 }
 
+- (void)removeLastViewFromArrangedSubview
+{
+    if([self.stackView.subviews count]) {
+        UIView *view = [self getLastSubview];
+        if (view) {
+            [self.stackView removeArrangedSubview:view];
+            [view removeFromSuperview];
+        }
+    }
+    
+}
+- (UIView *)getLastSubview
+{
+    UIView *view = nil;
+    if([self.stackView.subviews count]) {
+        view = self.stackView.subviews[[self.stackView.subviews count] - 1];
```

Comment:
>[self.stackView.subviews count] - 1]; [](start = 39, length = 37)

Nit: To aid in readablity, consider using a separate local variable that stores the lastIndex separately and then reference that variable here

Reply:
thanks

---
In reply to: [275516274](https://github.com/Microsoft/AdaptiveCards/pull/2675#discussion_r275516274) [](ancestors = 275516274)